rspace/src/test/scala/coop/rchain/rspace/history/TrieStructureTests.scala

```diff
@@ -78,44 +69,44 @@ class TrieStructureTests
     }
 
   it should "have two levels after inserting one element" in {
-    withTestTrieStore { (store, branch) =>
+    withTestTrieStore { implicit store => branch =>
```

Comment:
Oh that's how you do that?  That's hideous.

(No need to change anything, just saying...)

Reply:
What exactly is hideous?